File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.MoreEx_PB-JS-Nested-Conditional-Statements/P07_SchoolCamp.js
function schoolCamp(input) {

    let seasonType = input[0];
    let groupType = input[1];
    let studentsNumber = Number(input[2]);
    let stayNumber = Number(input[3]);

    let stayCost = 0.0;
    let sportType = "";

    switch (seasonType) {
        case "Winter":
            if (groupType === "boys") {
                sportType = "Judo";
                stayCost = stayNumber * 9.6;
            } else if (groupType === "girls") {
                sportType = "Gymnastics";
                stayCost = stayNumber * 9.6;
            } else {
                sportType = "Ski";
                stayCost = stayNumber * 10;
            }
            break;
        case "Spring":
            if (groupType === "boys") {
                sportType = "Tennis";
                stayCost = stayNumber * 7.2;
            } else if (groupType === "girls") {
                sportType = "Athletics";
                stayCost = stayNumber * 7.2;
            } else {
                sportType = "Cycling";
                stayCost = stayNumber * 9.5;
            }
            break;
        case "Summer":
            if (groupType === "boys") {
                sportType = "Football";
                stayCost = stayNumber * 15;
            } else if (groupType === "girls") {
                sportType = "Volleyball";
                stayCost = stayNumber * 15;
            } else {
                sportType = "Swimming";
                stayCost = stayNumber * 20;
            }
            break;
    }
    if (studentsNumber >= 50) {
        stayCost = stayCost * 0.5;
    } else if (studentsNumber >= 20) {
        stayCost = stayCost * 0.85;
    } else if (studentsNumber >= 10) {
        stayCost = stayCost * 0.95;
    }
    console.log(`${sportType} ${(stayCost * studentsNumber).toFixed(2)} lv.`);
}

schoolCamp(["Spring",
    "girls",
    20,
    7
]);

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03AdditionalProblems_ConditionalStatementsAdvanced/P07_SchoolCamp.java
package P03AdditionalProblems_ConditionalStatementsAdvanced;

import java.util.Scanner;

public class P07_SchoolCamp {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String season = scanner.nextLine();
        String groupType = scanner.nextLine();
        int numberStudents = Integer.parseInt(scanner.nextLine());
        int numberStay = Integer.parseInt(scanner.nextLine());
        double price = 0;
        String sportType = "";

        switch (season) {
            case "Winter":
                switch (groupType) {
                    case "girls":
                        price = numberStudents * numberStay * 9.6;
                        sportType = "Gymnastics";
                        break;
                    case "boys":
                        price = numberStudents * numberStay * 9.6;
                        sportType = "Judo";
                        break;
                    case "mixed":
                        price = numberStudents * numberStay * 10.0;
                        sportType = "Ski";
                        break;
                }
                break;
            case "Spring":
                switch (groupType) {
                    case "girls":
                        price = numberStudents * numberStay * 7.2;
                        sportType = "Athletics";
                        break;
                    case "boys":
                        price = numberStudents * numberStay * 7.2;
                        sportType = "Tennis";
                        break;
                    case "mixed":
                        price = numberStudents * numberStay * 9.5;
                        sportType = "Cycling";
                        break;
                }
                break;
            case "Summer":
                switch (groupType) {
                    case "girls":
                        price = numberStudents * numberStay * 15.0;
                        sportType = "Volleyball";
                        break;
                    case "boys":
                        price = numberStudents * numberStay * 15.0;
                        sportType = "Football";
                        break;
                    case "mixed":
                        price = numberStudents * numberStay * 20.0;
                        sportType = "Swimming";
                        break;
                }
                break;
        }

        if (numberStudents >= 50) {
            price = price * 0.5;
        } else if (numberStudents >= 20) {
            price = price * 0.85;
        } else if (numberStudents >= 10) {
            price = price * 0.95;
        }
        System.out.printf("%s %.2f lv.", sportType, price);
    }
}
